src/es6-module-bundler.ts

```diff
@@ -62,50 +62,37 @@ export async function bundle(
  *     export {$moduleA, $moduleB, $moduleBDefault};
  */
 async function prepareBundleModule(
```

Comment:
It looks like this entire block is actually indentation/formatting change due to clang-format or something IDK.